.github/ISSUE_TEMPLATE/icon_request.yml

```diff
@@ -1,37 +1,60 @@
-name: Icon Request
-description: Request a new icon, or changes to an existing icon
-title: '[ICON REQUEST]: <icon name>'
-labels: ["request:icon"]
-body:
-  - type: markdown
-    attributes:
-      value: "## Before you continue, please search our open/closed issues to see if a similar issue has been addressed."
-
-  - type: checkboxes
-    attributes:
-      label: I have searched through the issues and didn't find my problem.
-      options:
-        - label: Confirm
-          required: true
-
-  - type: input
-    id: about
-    attributes:
-      label: About the icon
-      description: Short description of why you think this icon belongs in our project.
-    validations:
-      required: true
-
-  - type: textarea
-    id: links
-    attributes:
-      label: Links and sources
-      description: Provide links to the icon's official website/repository/anywhere that shows what the technology is about, and its official logo. If available, also provide some resources where the icon/SVG can be found (Font Awesome, Icomoon, etc.).
-    validations:
-      required: true
-
-  - type: textarea
-    id: extrainformation
-    attributes:
-      label: Additional information
-      description: Is there anything else we should know about this icon?
+name: Icon Request
+description: Request a new icon, or changes to an existing icon
+title: '[ICON REQUEST]: icon name'
+labels: ["request:icon"]
+body:
+  - type: markdown
+    attributes:
+      value: "## Before you continue, please search our open/closed issues to see if a similar issue has been addressed."
+
+  - type: checkboxes
+    attributes:
+      label: I have searched through the issues and didn't find my problem.
+      options:
+        - label: Confirm
+          required: true
+
+  - type: input
+    id: about
+    attributes:
+      label: About the icon
+      description: Short description of why you think this icon belongs in our project.
+    validations:
+      required: true
+
+  - type: textarea
+    id: links
+    attributes:
+      label: Links and sources
+      description: Provide links to the icon's official website/repository/anywhere that shows what the technology is about, and its official logo. If available, also provide some resources where the icon/SVG can be found (Font Awesome, Icomoon, etc.).
+    validations:
+      required: true
+
+  - type: textarea
+    id: icons
+    attributes:
+      label: Icon links and previews
+      description: | 
+        Provide previews and links to official icons (preferably SVG's). 
+        *If there are no offical icons, or if you feel unofficial ones should be used instead, please add them here.*
+        
+        Example:
+        ```md
+        #### jule-original
+        <img alt="Git" width="100px" src="https://raw.githubusercontent.com/julelang/resources/6b1f4e68a8258435d87f29b9b2206b20a321f54e/jule.svg" ></img>
+        https://github.com/julelang/resources/blob/main/jule.svg
+        ```
+      value: |
```

Comment:
Should we use `value` or `placeholder`? It seems they do the same thing. 🤔

Reply:
Value will populate the issue, while placeholder will not. 

Value is also editable, which I don't think placeholder is. That's the reason I chose value, so it can easily be replaced by the contributor